src/shipit_pipeline/shipit_pipeline/api.py

```diff
@@ -7,13 +7,21 @@
 import logging
 
 from shipit_pipeline.pipeline import get_runnable_steps, get_running_steps, refresh_pipeline_steps, start_steps,\
-    PipelineStep
-
+    PipelineStep, MockPipelineStep
+mock1 = MockPipelineStep("mock1", {"timeout": 2}, [], "pending", 2)
+mock2 = MockPipelineStep("mock2", {"timeout": 5}, ["mock1"], "pending", 5)
+mock3 = MockPipelineStep("mock3", {"timeout": 10}, ["mock2"], "pending", 10)
+mock4 = PipelineStep("mock4", "https://tset",{"timeout": 2}, [], "pending")
 log = logging.getLogger(__name__)
 
-PIPELINES = {
-
-}
+PIPELINES = { "pipelinetest":["running",{
```

Comment:
split current PIPELINES variable into (1) PIPELINES_SPECS and (2) PIPELINES_STATE

PIPELINES_SPECS would look like

```python
PIPELINES_SPECS = [
    {
        "uid": "pipelinetest",
        "steps": [
            {
                "uid": "mock1",
                "url": "http://localhost:8012/mock",
                "parameters": {
                    "timeout": 2,
                }
            },
            {
                "uid": "mock2",
                "url": "http://localhost:8012/mock",
                "parameters": {
                    "timeout": 5,
                },
               "requires": [
                    "mock1",
                ],
            },
            {
                "uid": "mock3",
                "url": "http://localhost:8012/mock",
                "parameters": {
                    "timeout": 10,
                },
                "requires": [
                    "mock2",
                ],
            },
        ],
    },
]
```

and PIPELINES_STATE would look like

```python
PIPELINES_STATE = {
    "pipelinetest": {
        "mock1": {
          "status": "completed",
          "status_message": "Mock1 completed.",
          "ouput": {    # output will be used to pass to next step, mock2 in this case
            "some": "data"
          }
        },
        "mock2": {
          "status": "running",
        }
    }
}
```